### PR TITLE
criminalip add

### PR DESCRIPTION
**Pull request by parkjunmin**
add

### CODE REVIEW

**Comment by parkjunmin** [comment]
criminalip_96.png add

File: README.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ The following OSINT resources have been implemented for each artifact type:
   - [AlienVault OTX](https://otx.alienvault.com/)
   - [ARIN](https://search.arin.net/rdap/)
   - [Bad Packets](https://badpackets.net/)
+  - [Criminalip](https://www.criminalip.io/)
```
Comment (mitchmoser) [inline README.md]: please place this in alphabetical order

File: README.md [hunk 2]
```diff
@@ -71,6 +72,7 @@ The following OSINT resources have been implemented for each artifact type:
 ### Domain
   - [Alexa](https://www.alexa.com/siteinfo)
   - [BlueCoat](http://sitereview.bluecoat.com/#/)
+  - [Criminalip](https://www.criminalip.io/)
```
Comment (mitchmoser) [inline README.md]: please place this in alphabetical order

File: chrome/background.js [hunk 3]
```diff
@@ -234,6 +241,13 @@ chrome.contextMenus.create({
     "contexts": ["selection", "link", "image", "video", "audio"]
 });
 
+chrome.contextMenus.create({
+    "id": "Criminalip IP",
```
Comment (mitchmoser) [inline chrome/background.js]: I believe this `id` should be `Criminalip Domain`

File: firefox/background.js [hunk 4]
```diff
@@ -28,6 +28,16 @@ browser.contextMenus.create({
     }
 });
 
+browser.contextMenus.create({
+    id: "Criminalip IP",
+    title: "Criminalip",
+    contexts: ["selection", "link", "image", "video", "audio"],
+    parentId: "IP",
+    icons: {
+        "48": "icons/criminalip_96.png"
```
Comment (mitchmoser) [inline firefox/background.js]: please resize and rename this icon to 48x48 pixels

**Review by mitchmoser — CHANGES_REQUESTED** [review]
@parkjunmin thank you for the PR! 
I have reviewed and left a few notes on some requested changes.
Once these changes are made, I can accept the PR, close #30, and move to pre-release testing.

**Review by parkjunmin — COMMENTED** [review]
Hi. @mitchmoser 

As you said, I sorted it alphabetically and corrected it. Commit, please :)

Thank you.

**Comment by mitchmoser** [comment]
@parkjunmin after further review, it looks like the `background.js` files for both Chrome & Firefox were not using the latest versions from the master branch which resulted in branch conflicts.

Please update your branch and resolve these conflicts so that I can merge the PR.

File: README.md (hunk 1 above)
Comment (parkjunmin) [reply README.md]: It's sorted alphabetically Please check it out. :)

File: README.md (hunk 2 above)
Comment (parkjunmin) [reply README.md]: It's sorted alphabetically Please check it out. :)

**Comment by parkjunmin** [comment]
@mitchmoser Sorted alphabetically.
Also, when I checked the contents you reviewed, I checked that there was a conflict because there were two duplicate Criminalipes, so I changed it to domain and requested a commit.

Please check
Thank you.

https://github.com/mitchmoser/sputnik/commit/71e27252606c48c0f66c0b5f5b1b7a390e88b2e6 👍

**Comment by mitchmoser** [comment]
@parkjunmin there is still a conflict because your fork is [6 commits behind](https://github.com/parkjunmin/sputnik/compare/master...mitchmoser:sputnik:master).
Please update and deconflict your repo so that I can accept the PR.

**Pull request closed without merging**